application/src/main/java/de/wuespace/telestion/project/daedalus2/TransformedMessage.java

```diff
@@ -0,0 +1,10 @@
+package de.wuespace.telestion.project.daedalus2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.wuespace.telestion.api.message.JsonMessage;
+
+public record TransformedMessage(
+		@JsonProperty
+		String zeug
+) implements JsonMessage {
```

Comment:
I think the defualt constructor for serializing is missing. I don't know if it is required in java 16. We have to test it.